_install-and-configure/plugins.md

```diff
@@ -303,6 +303,22 @@ _<sup>1</sup>Dashboard Notebooks was merged in to the Observability plugin with
 _<sup>2</sup>Performance Analyzer is not available on Windows._
 
 
+### Downloading plugin distributions for offline installation
+
+Each bundled plugin distribution can be downloaded and installed offline from a [zip file](#install-a-plugin-from-a-zip-file) using the official `plugins` repository URL template:
+
+```html
+https://artifacts.opensearch.org/releases/plugins/<plugin-name>/<version>/<plugin-name>-<version>.zip
+```
+
+The `<plugin-name>` correponds to the name of the bundled plugin (for example, `analysis-icu`). The `<version>` must match the version of the OpenSearch distribution (for example, `2.19.1`). 
+
+For example, use the following URL to download the `analysis-icu` bundled plugin distribution for OpenSearch version `2.19.1`:
+
+```
```

Comment:
you used ` ```html` above but only ` ``` ` here - intentionally?

Reply:
Yes. <html> makes the text within the tags stand out. Here there are no tags so it'll be plain either way.